deploy/smart_proxy_dynflow_core.service

```diff
@@ -8,6 +8,7 @@ Type=forking
 User=foreman-proxy
 PIDFile=/var/run/foreman-proxy/smart_proxy_dynflow_core.pid
 ExecStart=/usr/bin/smart_proxy_dynflow_core -d -p /var/run/foreman-proxy/smart_proxy_dynflow_core.pid
+EnvironmentFile=-/etc/sysconfig/smart_proxy_dynflow_core
```

Comment:
Is the `-` supposed to be there before the path?

Reply:
Yea it's a systemd thing, it means ignore errors in the file isn't there I think

Reply:
exactly